modules/bitgo/src/v2/coinFactory.ts

```diff
@@ -128,6 +128,7 @@ import {
   ZkethToken,
   Lnbtc,
   Tlnbtc,
+  SuiToken,
```

Comment:
nit: place it in alphabetical order like the rest

Reply:
Will update with the next PR